src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.ts

```diff
@@ -83,7 +84,23 @@ export class LocalGameWrapperComponent extends GameWrapper<string> implements Af
             this.endGame = true;
             if (gameStatus.winner.isPlayer()) {
                 this.winner = MGPOptional.of($localize`Player ${gameStatus.winner.value + 1}`);
+                const loser: Player = gameStatus.winner.getOpponent();
+                const loserValue: number = loser.value;
+                if (this.players[gameStatus.winner.value].equalsValue('human')) { // When human win
+                    if (this.players[loserValue].equalsValue('human')) {
+                        this.winnerMessage = $localize`${this.winner.get()} won`;
+                    } else {
+                        this.winnerMessage = $localize`You win`;
+                    }
```

Comment:
C'est pas le même temps ! Joueur X a gagné / tu gagnes

Reply:
Changé en 
won/a gagné
lost/a perdu !

Reply:
mensonge, ligne 93 = "You win", ligne 97 = You lose

Reply:
outdated